### PR TITLE
Document code of condor_ce_host_network_check

### PR DESCRIPTION
**Pull request by matyasselmeci**

### CODE REVIEW

File: src/condor_ce_host_network_check [hunk 1]
```diff
@@ -47,13 +79,24 @@ class struct_sockaddr_in6(Structure):
...
 class struct_ifaddrs(Structure):
-    pass
+    """struct ifaddrs from ifaddrs.h
+    See getifaddrs in manpage section 3
+
+    A linked list; each element describes one network interface.  The ifa_next
+    field is a pointer to the next entry in the list.
+
+    """
+# _fields_ set separately because we need union_ifa_ifu to exist first
```
Comment (bbockelm) [inline src/condor_ce_host_network_check]: struct_ifaddrs, not union_ifa_ifu.

File: src/condor_ce_host_network_check [hunk 2]
```diff
@@ -136,16 +236,47 @@ def iface_matches(network_iface, pattern):
     return False
 
 def is_local_addr(ipv4_addr):
+    """Return if an IPv4 address is on a local network, i.e. one of
+    - 192.168.0.0/16
+    - 172.16.0.0/16
```
Comment (bbockelm) [inline src/condor_ce_host_network_check]: Might as well note that 172.16.0.0 really should be a /12, but that's kinda hard to do in python...

File: src/condor_ce_host_network_check (hunk 1 above)
Comment (matyasselmeci) [inline src/condor_ce_host_network_check]: OK, then I'm confused. We don't set `_fields_` separately in any of the other structs first; what makes this one different?

Comment (matyasselmeci) [inline src/condor_ce_host_network_check]: Never mind, I got it. It's because it's a linked list.

**Pull request merged**